Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
```

Comment:
I put it here since I only wanted _ABAddressBookManager to know of this class -- which header should I move it to? _ABAddressBookManager's header, or should I create its own separate header?

---
In reply to: [76672379](https://github.com/Microsoft/WinObjC/pull/935#discussion_r76672379) [](ancestors = 76672379)